site/src/js/index.js

```diff
@@ -51,7 +51,8 @@ function handleTags() {
     }
 
     Array.from(contentEl.querySelectorAll('.item')).forEach(el => {
-        const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false);
+        const show = tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false) && !(!tags.includes("tag-expired") && el.classList.contains("tag-expired"));
```

Comment:
```suggestion
        const show = (tags.includes("tag-expired") && !el.classList.contains("tag-expired")) &&
        	tags.reduce((sho, tag) => sho || el.classList.contains(`tag-${tag}`), false);
```

Reply:
I committed this in c2f7ab4c7ee4875a13426b9067293e79734385de with a small modification: I used `||` instead of `&&` in the `expired` logic, as dictated by [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws).